### DISCUSSION TITLE
onHover not being called during drag

### DISCUSSION THREAD

**Discussion by Coddo-Python** [visgl/deck.gl#9073] [Q&A]
As the title says, I noticed that onHover isnt called when dragging. For my use case, I need access to the mouse position even while dragging. This is because I want the user to be able to use **right-click** to draw a temporary line by holding right click and dragging. How would I go about this?

**Comment by Pessimistress** [comment]
https://deck.gl/docs/developer-guide/interactivity#calling-the-picking-engine-directly
